### PR TITLE
Add development report for 22 May 2017

### PR DESCRIPTION
**Pull request by justincormack**
Signed-off-by: Justin Cormack <justin.cormack@docker.com>

cutemouse1

### CODE REVIEW

File: reports/2017-05-22.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+The tenth weekly development report. Feel free to send PRs if you want to add to these reports (or correct them).
+
+There is now a new [linuxkit/virtsock] repository that contains Go bindings and sample code for 
+Hyper-V sockets and virtio sockets.
+
+There will be a [SIG Security call on 24 May 2017](https://github.com/linuxkit/linuxkit/blob/master/reports/sig-security/2017-05-24.md).
+
+## Overview
+
+- We have mostly added a hub org override so you can do `make ORG=myrepo/myorg` when building packages, rather than pushing to the `linuxkit` org.
```
Comment (rneugeba) [inline reports/2017-05-22.md]: `myrepo/myorg` -> `myorg/myrepo`

File: reports/2017-05-22.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+- ongoing work has continued to move all builds to use the same Alpine base package. [linuxkit/linuxkit#1856] [@rneugeba]
+- Label support - images may now specify an `org.mobyproject.config` label which contains a JSON version of the config. Anything set in this config will be the default, but can be overridden by the yaml file. In many cases this means that the yaml is nor needed, simplifying the config a lot. [linuxkit/linuxkit#1862] [@justincormack]
+
+## PRs merged
+
+Kernel and Base:
+
+- Update Hyper-V patches for 4.11 and add Hub org override for kernel builds [linuxkit/linuxkit#1844] [@GordonTheTurtle] [@justincormack] [@rneugeba]
```
Comment (rneugeba) [inline reports/2017-05-22.md]: remove `@GordonTheTurtle`?

File: reports/2017-05-22.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+- kernel: Add vmlinux to debug builds [linuxkit/linuxkit#1849] [@rneugeba]
+- Update containerd [linuxkit/linuxkit#1861] [@rneugeba]
```
Comment (rneugeba) [inline reports/2017-05-22.md]: that was you not me

File: reports/2017-05-22.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
+# Development report for 22 May 2017
+
+The tenth weekly development report. Feel free to send PRs if you want to add to these reports (or correct them).
+
+There is now a new [linuxkit/virtsock] repository that contains Go bindings and sample code for 
+Hyper-V sockets and virtio sockets.
+
+There will be a [SIG Security call on 24 May 2017](https://github.com/linuxkit/linuxkit/blob/master/reports/sig-security/2017-05-24.md).
+
+## Overview
```
Comment (rneugeba) [inline reports/2017-05-22.md]: I would add a line that we have significantly improved testing and continue adding more tests

File: reports/2017-05-22.md (hunk 1 above)
Comment (justincormack) [inline reports/2017-05-22.md]: I meant not docker.io by repo, not sure what the correct term is

File: reports/2017-05-22.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
+# Development report for 22 May 2017
+
+The tenth weekly development report. Feel free to send PRs if you want to add to these reports (or correct them).
+
+There is now a new [linuxkit/virtsock] repository that contains Go bindings and sample code for 
+Hyper-V sockets and virtio sockets.
+
+There will be a [SIG Security call on 24 May 2017](https://github.com/linuxkit/linuxkit/blob/master/reports/sig-security/2017-05-24.md).
```
Comment (riyazdf) [inline reports/2017-05-22.md]: Could we add a line that links to the main SIG [description](https://github.com/linuxkit/linuxkit/blob/master/sigs/security/README.md) and says that this is scheduled to repeat every other week?

File: reports/2017-05-22.md (hunk 1 above)
Comment (rneugeba) [inline reports/2017-05-22.md]: yes, but "`linuxkit` org" is confusing.

Maybe:
"We have mostly added support for overriding registry and/or hub org when  building packages by specifying the `ORG` make variable."

**Pull request merged**